scss/_base_hr.scss

```diff
@@ -20,7 +20,33 @@
 
     // stylelint-disable-next-line selector-no-qualifying-type
     &.is-fixed-width {
-      @extend %fixed-width-container;
+      margin-left: auto;
```

Comment:
As literally every single selector in this file uses `stylelint-disable-next-line selector-no-qualifying-type` it would make sense to disable it for the whole file.

Just use `// stylelint-disable selector-no-qualifying-type` on top of the file but don't forget to enable it using `// stylelint-enable selector-no-qualifying-type` at the end.

Reply:
done